packages/webapp/pages/log/index.tsx

```diff
@@ -44,11 +43,11 @@ import CardArchetypeReveal from '../../components/log/CardArchetypeReveal';
 import CardShare from '../../components/log/CardShare';
 import CardNoData from '../../components/log/CardNoData';
 
-export const getServerSideProps: GetServerSideProps = async () => {
-  return {
-    notFound: true,
-  };
-};
+// export const getServerSideProps: GetServerSideProps = async () => {
```

Comment:
This intentional?

Reply:
Just to get Tomer's approval that bug was fixed then I revert